generator/src/OperatorClassGenerator.php

```diff
@@ -78,9 +78,10 @@ public function createClass(GeneratorDefinition $definition, OperatorDefinition
                 $constuctor->addComment('@param ' . $type->doc . ' ...$' . $argument->name . rtrim(' ' . $argument->description));
 
                 if ($argument->variadicMin > 0) {
+                    $namespace->addUse(InvalidArgumentException::class);
                     $constuctor->addBody(<<<PHP
                     if (\count(\${$argument->name}) < {$argument->variadicMin}) {
-                        throw new \InvalidArgumentException(\sprintf('Expected at least %d values for \${$argument->name}, got %d.', {$argument->variadicMin}, \count(\${$argument->name})));
+                        throw new InvalidArgumentException(\sprintf('Expected at least %d values for \${$argument->name}, got %d.', {$argument->variadicMin}, \count(\${$argument->name})));
```

Comment:
Do you mean to throw a `MongoDB\Exception\InvalidArgumentException` here (based on the current import) or a generic, non-vendored exception? I suppose it doesn't really matter, but this diff dropped the leading `\` on the class name.

Reply:
Yes I want to throw the MongoDB exception.